librz/analysis/arch/arm/arm_il32.c

```diff
@@ -2589,7 +2596,6 @@ static RzILOpEffect *write_reg_lane(arm_reg reg, ut32 lane, ut32 vec_size, RzILO
  */
 static RzILOpEffect *vmov(cs_insn *insn, bool is_thumb) {
 	if (OPCOUNT() < 2 || !ISREG(0) || (!ISIMM(1) && !ISREG(1))) {
-		// for now, only support vmov rt, rn
```

Comment:
Not anymore? But the code hasn't changed

Reply:
yep, it's a comment left by florian when create arm32 lifter. can be removed now.